src/codejar.ts

```diff
@@ -13,7 +13,7 @@ type Position = {
   dir?: "->" | "<-"
 }
 
-export function CodeJar(editor: HTMLElement, highlight: (e: HTMLElement) => void, opt: Partial<Options> = {}) {
+export const CodeJar = (editor: HTMLElement, highlight: (e: HTMLElement) => void, opt: Partial<Options> = {}) => {
```

Comment:
Why this change?

Reply:
This is my fault, i commit this even if i don't want... I'm so sorry